src/autopas/selectors/tuningStrategy/ruleBasedTuning/RuleLanguage.g4

```diff
@@ -0,0 +1,143 @@
+// This file describes the grammar of the rule language used by rule based tuning. The parsing code is generated using
+// the ANTLR plugin for CLion. It is generated into parser_generated/ and committed to the repository. Changing this
+// file requires manual regeneration of the code into this directory, and also committing (and formatting) the generated
+// files.
+
+grammar RuleLanguage;
```

Comment:
this is another file we need to maintain whenever we add any new config element. Could we reasonably generate this via some script or mini c++ program like the `zsh` completions`? Otherwise this has to be mentioned in the contribution guidelines.

Reply:
I will think a bit if we can do it easily. Note that the generated code files are not generated automatically from this grammar at the moment. It is a manual process, so regenerating them manually after any change is required.